src/containers/tud/schemas/PreSurveySchema.js

```diff
@@ -17,7 +16,7 @@ const {
   TYPICAL_DAY_FLAG
 } = PROPERTY_CONSTS;
 
-const schema = {
+const createSchema = (trans :(string, ?Object) => string) => ({
```

Comment:
I've seen you define `trans :(string, ?Object) => string` in many places. Please create a type for it and reuse it.

Also, why does this `trans` return a string but the `trans` in my other comment returns `Object`?

Reply:
should be object